galite-core/src/main/kotlin/org/kopi/galite/visual/ApplicationConfiguration.kt

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2013-2020 kopiLeft Services SARL, Tunis TN
+ * Copyright (c) 1990-2020 kopiRight Managed Solutions GmbH, Wien AT
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1 as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+package org.kopi.galite.visual
+
+import org.kopi.galite.util.Rexec
+import java.io.File
+
+/**
+ * Manages Application configuration data
+ */
+abstract class ApplicationConfiguration {
+  // --------------------------------------------------------------
+  //   Application Properties
+  // --------------------------------------------------------------
+  /**
+   * Property app.version
+   * Returns the version of the application
+   */
+  abstract val version: String
+
+  /**
+   * Property app.name
+   * Returns the application name
+   */
+  abstract val applicationName: String
+
+  /**
+   * Property app.comment
+   * Returns the information text about this application
+   */
+  abstract val informationText: String
```

Comment:
Change to abstract methods instead of abstract fields ( check other fields too )